README.md

```diff
@@ -1,2 +1,64 @@
-# First-capstone-Project
-This project shows my acquired knowledge so far on Html, Css and Javascript
+# First-capstone-Project: Milestone 6
+
+This project shows my acquired knowledge so far;
+
+> The capstone project demonstrates Javascript, Html and Css.
+> This is also a project that shows my proficiency as I go through Microverse. It is a work-in-progress and I hope to make it a masterpiece.
+
+## Live Demo
+
+[Link to Live Demo](https://eternalgratis.github.io/First-capstone-Project/)
```

Comment:
- [ ] Your live demo is redirecting to the README file and that is because the index.html file is not in the root of your project